### ISSUE TITLE
Allocate+ Column Headers

### ISSUE THREAD

**Issue by applebyter** [UQTools/Queue#1]
(Not a bug in queue necessarily - but assumptions about Allocate+ appear to no longer be valid.)

It appears that for 2023, when you select **ACTIVITY_CODE** as a desired field in the Allocate+ report, the file generated does not include this as a column header - instead it names the column "**SHORT_CODE**". The queue system rejects this file and one has to rename the column in the spreadsheet to **ACTIVITY_CODE** for it to be accepted.

Copied from Peter Sutton's issue raised in CSSE1001.

**Comment by pipw** [comment]
`ACTIVITY_CODE` is hard-coded into the source files (in multiple places) as are `STUDENT_CODE` and `ACTIVITY_GROUP_CODE`.

Simple find and replace strategy, while not ideal, might solve the current issue from a development perspective. Operations is another story!

Risks:
* code base hasn't been touched in 2 years (3rd party decencies out of date)
* last deployment to PROD was also 2 years (Infrastructure changes)
* code base has no comments
* the fact of the same strings being hard-coded into multiple files hints at code smells and technical debt.
* original developer availability

@applebyter I'm assuming that this tool remains fundamental to ITEE course delivery. Is the original dev still involved with the ongoing maintenance of the code-base? While the fix itself looks to be simple enough, it might be an opportunity to upgrade 3rd party dependencies and deploy to newer zone images.

**Comment by applebyter** [comment]
@pipw Yes, this tool is heavily used by several large courses in ITEE. The original developer is typically not available for maintenance.

Upgrading 3rd party dependencies, deploying to a new zone, and replacing hard-coded literal values with symbolic constants could be beneficial.

This issue is the lowest priority of the issues for this tool, but may be simple enough to be worth fixing.
